nixos/doc/manual/development/running-nixos-tests-interactively.section.md

```diff
@@ -4,19 +4,19 @@ The test itself can be run interactively. This is particularly useful
 when developing or debugging a test:
 
 ```ShellSession
-$ nix-build nixos/tests/login.nix -A driverInteractive
+$ nix build -f . nixosTests.login.driverInteractive
```

Comment:
Isn't nix-command still an experimental feature?

Reply:
You're right, changed to use old style